tests/unit/Resolution.spec.ts

```diff
@@ -27,8 +27,7 @@ describe('Special Resolution Form component', () => {
       legalType: 'CP'
     }
   }
-
-  beforeAll(() => {
```

Comment:
This was painful, would make some unit tests fail when ran in batches versus running a single unit test alone.

Reply:
I guess some data needed to be re-initialized between tests.

Glad you figured it out.